### ISSUE TITLE
each_page not working?

### ISSUE THREAD

**Issue by steveklebanoff** [thierryxing/sentry-ruby-api#1]
I'm getting an error when I try to use `.each_page` -- it seems as though when there is more than 1 page the object that is yielded is the string of the next page URL instead of a ruby object containing objects from the page.  Do you see this on your side too?

**Comment by steveklebanoff** [comment]
here's my janky makeshift solution for getting all issues from a project, am I missing something?

```ruby
issues = []

current_page = client.project_issues(project_name, query: query)
loop do
  issues += current_page
  if current_page.has_next_page?
    # remove URL prefix so client can handle it
    next_page_path = current_page.next_page.gsub(client.endpoint, '')
    current_page = client.get(next_page_path)
  else
    break
  end
end
```

**Comment by thierryxing** [comment]
OK, I will check it as soon as possible. @steveklebanoff

**Comment by thierryxing** [comment]
@steveklebanoff. Sorry, this is really a mistake, and I released a new 0.3.3 version and fix this bug.
Now you can use each_page method correctly.
Thanks for reporting this issue.

**Comment by steveklebanoff** [comment]
cool, thanks for the fix!
